src/i18n/translations/en.yml

```diff
@@ -791,6 +791,7 @@ terminal.reducestrategy.recurring.help: "If checked, each time a new DCA is fill
 terminal.reducestrategy.persistent: "Persistent"
 terminal.reducestrategy.persistentrecurring: "Persistent/Recurring"
 terminal.reducestrategy.persistent.help: "If checked, it won't close the position if there are pending DCAs."
+terminal.reducestrategy.tpwarning: "This will replace your Take Profit Targets."
```

Comment:
@cwagner22  have you added these translations to other files too, I don't see them here.

Reply:
I was planning to update cs.yml with find and replace, which I did now. I think now we have the habit to put all the text strings in the translation file instead of hardcoding but good point.